test/powershell/Host/Startup.Tests.ps1

```diff
@@ -103,7 +103,7 @@ Describe "Validate start of console host" -Tag CI {
     }
```

Comment:
@adityapatwardhan, your last commit had 5 failures in `PowerShell-CI-macos`
Error occurred in test script '/Users/runner/runners/2.164.8/work/1/s/test/powershell/engine/Remoting/PSSession.Tests.ps1'
```powershell
At /Users/runner/runners/2.164.8/work/1/s/test/powershell/engine/Remoting/PSSession.Tests.ps1:79 char:14
+ ?            ($platformInfo.Platform -eq 'centos' -and $platformInfo.Ve ?
+               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The assignment expression is not valid. The input to an assignment operator must be an object that is able to accept assignments, such as a variable or a property.
at <ScriptBlock>, /Users/runner/runners/2.164.8/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.164.8/work/1/s/test/powershell/engine/Remoting/PSSession.Tests.ps1'
```powershell
At /Users/runner/runners/2.164.8/work/1/s/test/powershell/engine/Remoting/PSSession.Tests.ps1:79 char:14
+ ?            ($platformInfo.Platform -eq 'centos' -and $platformInfo.Ve ?
+               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The assignment expression is not valid. The input to an assignment operator must be an object that is able to accept assignments, such as a variable or a property.
at <ScriptBlock>, /Users/runner/runners/2.164.8/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.164.8/work/1/s/test/powershell/engine/Remoting/PSSession.Tests.ps1'
```powershell
At /Users/runner/runners/2.164.8/work/1/s/test/powershell/engine/Remoting/PSSession.Tests.ps1:79 char:14
+ ?            ($platformInfo.Platform -eq 'centos' -and $platformInfo.Ve ?
+               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The assignment expression is not valid. The input to an assignment operator must be an object that is able to accept assignments, such as a variable or a property.
at <ScriptBlock>, /Users/runner/runners/2.164.8/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/runner/runners/2.164.8/work/1/s/test/powershell/engine/Remoting/PSSession.Tests.ps1'
```powershell
At /Users/runner/runners/2.164.8/work/1/s/test/powershell/engine/Remoting/PSSession.Tests.ps1:79 char:14
+ ?            ($platformInfo.Platform -eq 'centos' -and $platformInfo.Ve ?
+               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
The assignment expression is not valid. The input to an assignment operator must be an object that is able to accept assignments, such as a variable or a property.
at <ScriptBlock>, /Users/runner/runners/2.164.8/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Test PSDesiredStateConfiguration.Error occurred in Describe block
```powershell
The term 'Get-Platform' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At /Users/runner/runners/2.164.8/work/1/s/test/powershell/Modules/PSDesiredStateConfiguration/PSDesiredStateConfiguration.Tests.ps1:31 char:21
```